about.html

```diff
@@ -38,7 +40,7 @@
             <section class="nav-bar d-flex d-none">
                 <p class="logo">Architects</p>
                 <ul class="nav-list d-flex">
-                    <li><a href="/index.html">Home</a></li>
+                    <li><a href="index.html">Home</a></li>
                     <li><a href="">Prizes</a></li>
                     <li><a href="">Biography</a></li>
                     <li><a href="">Projects</a></li>
```

Comment:
- [ ] Your page allows us to scroll x which doesn't look nice for the UX(user experience). Let us fix this by adding scroll-x: hidden inside the body in our CSS. for more check this image.

![main3](https://user-images.githubusercontent.com/55635977/128430194-1d148221-4b7e-49c8-9eff-beaaa814d072.png)

- [ ] You can also check for the design to implement the top dark bar with the two icons, logout, English and My page as suggested.

Reply:
Done, thx for the feedback!